src/Tests/Microsoft.Diagnostics.Monitoring.UnitTestApp/Scenarios/FunctionProbes/PerFunctionProbeProxy.cs

```diff
@@ -59,6 +70,16 @@ public int GetProbeInvokeCount(MethodInfo method)
             return probe.GetInvokeCount();
         }
 
+        public XunitException GetProbeAssertException(MethodInfo method)
```

Comment:
`bool TryGetProbeAssertException(MethodInfo method, out XunitException exception)`

This will be more friendly with nullability when we enable that in the future and forces the caller to think about whether the returned exception was actually provided or not.